distribution/test/itests/test-itests-common/src/main/java/org/codice/alliance/test/itests/common/AbstractAllianceIntegrationTest.java

```diff
@@ -0,0 +1,101 @@
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.test.itests.common;
+
+import static org.ops4j.pax.exam.CoreOptions.junitBundles;
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.CoreOptions.options;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.karafDistributionConfiguration;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.keepRuntimeFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.lang.text.StrSubstitutor;
+import org.ops4j.pax.exam.Option;
+import com.google.common.collect.ImmutableMap;
+
+import ddf.common.test.AbstractIntegrationTest;
+
+public abstract class AbstractAllianceIntegrationTest extends AbstractIntegrationTest {
+
+    public static final String[]
+            DEFAULT_ALLIANCE_APPS = {"catalog-app", "solr-app", "spatial-app", "security-app"};
+
+    @Override
+    protected Option[] configureDistribution() {
+        return options(karafDistributionConfiguration(maven().groupId(
+                "org.codice.alliance.distribution")
+                .artifactId("alliance")
+                .type("zip")
+                .versionAsInProject()
+                .getURL(), "alliance", KARAF_VERSION).unpackDirectory(new File("target/exam"))
+                .useDeployFolder(false));
+    }
+
+    @Override
+    protected Option[] configureStartScript() {
+        //add test dependencies to the test-dependencies-app instead of here
+        return options(junitBundles(),
+
+                features(maven().groupId("org.codice.alliance.test.itests")
+                        .artifactId("test-itests-dependencies-app")
+                        .type("xml")
+                        .classifier("features")
+                        .versionAsInProject(), "alliance-itest-dependencies"),
+
+                features(maven().groupId("ddf.distribution")
+                        .artifactId("sdk-app")
+                        .type("xml")
+                        .classifier("features")
+                        .versionAsInProject()));
+    }
+
+    @Override
+    protected Option[] configureCustom() {
+        return options(keepRuntimeFolder());
```

Comment:
I left it because that seems to be the preference on this project, all the itests had it explicitly declared. I know of other projects that do this as well. I have no preference, I don't mind removing it